app/build.gradle

```diff
@@ -32,7 +32,7 @@ android {
   defaultConfig {
     applicationId 'net.skyscanner.backpack'
     versionCode 1
-    versionName "1.0.0"
```

Comment:
now that we just have 1 version again we can use the version name for the demo app again